frontend/src/Editor/LeftSidebar/index.jsx

```diff
@@ -225,7 +223,7 @@ export const LeftSidebar = forwardRef((props, ref) => {
       <LeftSidebarInspector
         darkMode={darkMode}
         selectedSidebarItem={selectedSidebarItem}
-        currentState={currentState}
+        setSelectedSidebarItem={handleSelectedSidebarItem}
```

Comment:
Don't think this is being used in the component. Please check and remove it if not needed

Reply:
Not used, removed.